src/describe/src/describe/compute.rs

```diff
@@ -0,0 +1,404 @@
+use polars::prelude::*;
+use tabled::{builder::Builder, Table};
+
+use crate::Args;
+
+#[derive(Debug, Clone, PartialEq)]
+struct Analyze {
+	name: String,
+	dtype: DataType,
+
+	min: Option<f64>,
+	max: Option<f64>,
+	mean: Option<f64>,
+	median: Option<f64>,
+	q1: Option<f64>,
+	q3: Option<f64>,
+	std: Option<f64>,
+	sum: Option<f64>,
+}
+
+type TableRecord<'s> = [&'s str; 9];
+
+pub fn compute(df: DataFrame, args: &Args) -> PolarsResult<(Table, Vec<DataType>)> {
+	let mut types = Vec::with_capacity(df.width());
+
+	let mut builder = Builder::default();
+
+	builder.push_record(Analyze::HEADERS);
+
+	for series in df.get_columns() {
+		let analyze = Analyze::from(series);
+
+		let name = truncate(&analyze.name, 10);
+
+		let record: TableRecord = [
+			&name,
+			&analyze.dtype.to_string(),
+			&to_string(analyze.min, args),
+			&to_string(analyze.max, args),
+			&to_string(analyze.mean, args),
+			&to_string(analyze.median, args),
+			&to_string(analyze.q1, args),
+			&to_string(analyze.q3, args),
+			&to_string(analyze.std, args),
+		];
+		builder.push_record(record);
+
+		types.push(analyze.dtype);
+	}
+
+	Ok((builder.build(), types))
+}
+
+impl From<Series> for Analyze {
+	fn from(series: Series) -> Self {
+		Analyze::from(&series)
+	}
+}
+
+impl From<&Series> for Analyze {
+	fn from(series: &Series) -> Self {
+		let name = series.name().to_owned();
+		let dtype = series.dtype().to_owned();
+
+		let mut ret = Self {
+			name,
+			dtype,
+
+			min: None,
+			max: None,
+			mean: None,
+			median: None,
+			q1: None,
+			q3: None,
+			std: None,
+			sum: None,
+		};
+
+		if !ret.dtype.is_numeric() {
+			return ret;
+		}
+
+		let mut arr = series
+			.cast(&DataType::Float64)
+			.expect("could not cast series to f64")
+			.f64()
+			.expect("could not extract series as f64 iterator")
+			.into_iter()
+			.filter(|x| x.is_some())
+			.map(|x| x.unwrap())
+			.collect::<Vec<f64>>();
+
+		arr.sort_by(|a, b| a.total_cmp(b));
+
+		let Some(first) = arr.first().copied() else {
+			return ret;
+		};
+
+		let mut min = first;
+		let mut max = first;
+		let mut sum = first;
+
+		for value in arr.iter().skip(1) {
+			min = min.min(*value);
+			max = max.max(*value);
+			sum += value;
+		}
+
+		let mean = sum / arr.len() as f64;
+		let median = if arr.len() % 2 == 0 {
+			let mid = arr.len() / 2;
+			(arr[mid - 1] + arr[mid]) / 2.0
+		} else {
+			arr[arr.len() / 2]
+		};
```

Comment:
```suggestion
		let median = arr[((arr.len() as float) / 2).floor()]
```

Reply:
`polars` `.median()` use the average if the len is odd

#### example:
- `median` of `[2, 4]` is `3`